src/_ExtendedView.scss

```diff
@@ -0,0 +1,129 @@
+$blue: #3eb4e0;
+
```

Comment:
Based on your HTML, there are likely parts of your css that you could refactor to take advantage of nesting with sass